VILIB/Controllers/FaceDetectionController.cs

```diff
@@ -16,12 +19,13 @@ namespace VILIB.Controllers
     [EnableCors(origins: "*", headers: "*", methods: "*")]
     public class FaceDetectionController : ApiController
     {
+        private const int _grayFaceImageSize = 100;
```

Comment:
It would be good to put these constants into .config file

Reply:
Removed constants from c# code with latest commit, although there are some left in javascript files